weblate/templates/mail/account_activity.html

```diff
@@ -4,7 +4,7 @@
 
 {% block content %}
 <p>
-{% blocktrans %}There has been a following change in your account at {{ site_title }}:{% endblocktrans %}
```

Comment:
This actually means a change occurring after something.
Yurchor already fixed one grave error, and the third I disclosed already.